Update index.html

a-autotte opened the pull request:
Français : Ceci est pour le changement des noms des membres du conseil suite au vote de la réunion sur le discord

English : This is for the change of the names for the member of council after voting in the meeting on the discord

Commit:
Update index.html

Français : Ceci est pour le changement des noms des membres du conseil suite au vote de la réunion sur le discord

English : This is for the change of the names for the member of council after voting in the meeting on the discord

obrassard reviewed (changes requested):
Il faudrait modifier les photos associés au nom pour que ça fit avec les personnes concerné.
Pour les nouvelles personnes, essayer d'obtenir une photo, sinon en dernier recours utiliser `img/users/empty.jpg`

ghost commented:
> Il faudrait modifier les photos associés au nom pour que ça fit avec les personnes concerné.
> Pour les nouvelles personnes, essayer d'obtenir une photo, sinon en dernier recours utiliser `img/users/empty.jpg`

Je comprends entièrement Brassard c'est ce que j'allais faire et ce que je vais en ce moment

Commit:
Update index.html

Français : Modification des photos des membres et ajout de fichiers jpeg

English : Updating photos of members and adding jpeg files

TheAutismAlgoTrader committed:
Français : Ajout de fichiers jpeg de membres du conseil English : Adding jpeg files of council members

TheAutismAlgoTrader committed:
Merge branch 'patch-1' of https://github.com/a-autotte/adeptinfo.ca into patch-1

obrassard commented:
Duplicate of #9
